docs/plugins/convert.rst

```diff
@@ -140,3 +140,52 @@ and the given command is used for all conversions.
     convert:
         command: ffmpeg -i $source -y -vn -aq 2 $dest
         extension: mp3
+
+
+Gapless MP3 encoding
+````````````````````
+
+Due to limitations in the ``ffmpeg`` encoder, by default the ``convert`` plugin
+does not produce MP3s with accurate length tags (it does not write a LAME MP3
+info tag). This means that the MP3s are not "`gapless`_", and can result in
+noticeable gaps between tracks during playback.
+
+.. _gapless: http://wiki.hydrogenaud.io/index.php?title=Gapless_playback
+
+To work around this problem, you must use the `LAME`_ MP3 encoder, which will
+produce MP3s with accurate length tags.
+
+.. _LAME: http://lame.sourceforge.net/
+
+To do this on Linux, you need to install ``lame``, create a script (e.g.
+``/home/user/.config/beets/mp3.sh``) with the following contents, and make it
+executable.
+
+::
+
+    #!/bin/sh
+    /usr/bin/ffmpeg -i "$1" -f wav - | /usr/bin/lame -V 2 --noreplaygain - "$2"
```

Comment:
Maybe don't put the absolute paths which may vary :

    ffmpeg -i "$1" -f wav - | lame -V 2 --noreplaygain - "$2"